azure/template.json

```diff
@@ -39,8 +39,11 @@
         },
         "storageKind": {
             "type": "string"
+        },
+        "functionsExtensionVersion": {
+            "type": "string",
+            "defaultValue": "~2.0"
```

Comment:
Is the default correct?  I've only ever seen the value for this setting as ~1, ~2, etc (so no decimal).  Did a quick search of SFA GitHub and couldn't see any examples using a decimal point

Reply:
Yes. It's set to that so we can opt-out of the Azure Functions platform upgrade to v3. We're pinning any apps with ~2 to ~2.0.